schema.xsd

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           elementFormDefault="qualified" attributeFormDefault="unqualified">
+    <xs:element name="release">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="changelog" maxOccurs="1" minOccurs="1">
+                    <xs:complexType>
+                        <xs:attribute name="href" type="xs:anyURI" use="required"  />
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="whatsNew" minOccurs="1" maxOccurs="unbounded">
+                    <xs:complexType>
+                        <xs:sequence>
+                            <xs:element name="regular" type="whatsNewInfo" minOccurs="1" maxOccurs="1" />
+                            <xs:element name="admin" type="whatsNewInfo" minOccurs="0" maxOccurs="1" />
+                        </xs:sequence>
+                        <xs:attribute name="lang" type="xs:language" use="required" />
+                    </xs:complexType>
+                </xs:element>
+            </xs:sequence>
+            <xs:attribute name="version" type="version" use="required" />
+        </xs:complexType>
+    </xs:element>
+
+    <xs:simpleType name="version">
+        <xs:restriction base="xs:string">
+            <xs:pattern value="(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)"/>
+            <xs:minLength value="6"/>
+            <xs:maxLength value="12"/>
+        </xs:restriction>
+    </xs:simpleType>
+
+    <xs:complexType name="whatsNewInfo">
+        <xs:sequence>
+            <xs:element name="item" minOccurs="1" maxOccurs="3">
+                <xs:simpleType>
+                    <xs:restriction base="xs:string">
+                        <xs:maxLength value="100"/>
```

Comment:
That one is tricky, because translations could be significantly longer. Just thinking of Spanish and German.

Reply:
Also is there a reason to restrict this? Like we decide what gets in there right?

Reply:
The idea is that the bullet points should be short and crispy. The 100 chars was in the discussion, after double checking in the design issue we did not specify a hard number though. So, sure I can lift this restriction and instead call for a short phrasing. At least about the original text we have control, so that should be OK.

Reply:
The minLength is also arbitrary, let me lower this to 1 to avoid empty lines. Not that this cannot be tricked, but there's not risk with that either.